packages/indexer-agent/src/agent.ts

```diff
@@ -1020,21 +1027,13 @@ export class Agent {
       eligibleAllocations.map(allocation => allocation.subgraphDeployment.id),
     )
 
-    // Log details if active deployments are different from target deployments
-    const isReconciliationNeeded = !isEqual(
-      deploymentIDSet(activeDeployments),
-      deploymentIDSet(targetDeployments),
-    )
-    if (isReconciliationNeeded) {
-      // TODO: Return early in here case reconciliation is not needed
-      this.logger.debug('Reconcile deployments', {
-        syncing: activeDeployments.map(id => id.display),
-        target: targetDeployments.map(id => id.display),
-        withActiveOrRecentlyClosedAllocation: eligibleAllocationDeployments.map(
-          id => id.display,
-        ),
-      })
-    }
+    logger.debug('Reconcile deployments', {
```

Comment:
Removing the conditional?

Reply:
This conditional is used only for logging the inputs for deployment reconciliation. 

Having this information in every loop, even when there's no operation to perform, could be helpful as a way of providing evidence.

Reply:
If we remove the conditional, we could merge this log with the following one.
```

Reply:
Resolving this thread in favor of https://github.com/graphprotocol/indexer/pull/753#discussion_r1306105071